src/main/java/chatter/task/Event.java

```diff
@@ -0,0 +1,67 @@
+package chatter.task;
+
+import java.time.LocalDate;
+
+/**
+ * Represents a task that has a start and end time.
+ *
+ * @author Anthony Tamzil
+ * @version CS2103T Individual Project AY2023/24 Semester 1
+ */
+public class Event extends Task {
+    /** A string indicating start date / time of task */
+    protected String altStart;
+    /** A localDate indicating start date / time of task */
+    protected LocalDate start;
+    /** A string indicating end date / time of task */
+    protected String altEnd;
+    /** A localDate indicating end date / time of task */
+    protected LocalDate end;
+
+    /**
+     * A constructor to initialize the chatter.task.Event class.
+     *
+     * @param description Description of the task.
+     * @param start Start time of the task.
+     * @param end End time of the task.
+     */
+    public Event(String description, String start, String end) {
+        super(description);
+
+        try {
+            this.start = LocalDate.parse(start);
+            this.altStart = "";
+        } catch (Exception e) {
+            this.altStart = start;
+        }
+
+        try {
+            this.end = LocalDate.parse(end);
+            this.altEnd = "";
+        } catch (Exception e) {
+            this.altEnd = end;
+        }
+    }
+
+    /**
+     * Returns the string representation of the chatter.task.Event that will
+     * be displayed to the user in the list.
+     *
+     * @return The string representation of the chatter.task.Event object.
+     */
+    @Override
+    public String toString() {
+        return "[E]" + super.toString() + " (from: " + this.altStart + convertDateToString(this.start)
```

Comment:
Very good naming for function! convertDateToString allows me to know the role of the function right away!